letsencrypt/files/renew_letsencrypt_cert.sh.jinja

```diff
@@ -2,22 +2,8 @@
 {% from "letsencrypt/map.jinja" import letsencrypt with context %}
 COMMON_NAME="$1"
 
-JOINED=""
-for DOMAIN in "$@"
-do
-    JOINED+=" -d $DOMAIN"
-done
-
-if ! /usr/local/bin/check_letsencrypt_cert.sh "$@" > /dev/null
-then
-    {{ letsencrypt.cli_install_dir }}/letsencrypt-auto certonly $JOINED --non-interactive || exit 1
-    cat /etc/letsencrypt/live/${COMMON_NAME}/fullchain.pem \
-        /etc/letsencrypt/live/${COMMON_NAME}/privkey.pem \
-        > /etc/letsencrypt/live/${COMMON_NAME}/fullchain-privkey.pem || exit 1
-    chmod 600 /etc/letsencrypt/live/${COMMON_NAME}/fullchain-privkey.pem || exit 1
-    {%- if letsencrypt.post_renew.cmds %}
```

Comment:
I think the `post_renew` should still be there, right?

Reply:
Yep, looks like I removed that line by accident. What would be the proper way to get it back in? Change this pull request? Open a new one?

Reply:
@dmaphy can you push a new PR re-adding them?